RoaringBitmap/src/test/java/org/roaringbitmap/buffer/TestImmutableRoaringBitmap.java

```diff
@@ -1505,17 +1505,17 @@ public void testAbsentBits() {
               .toMutableRoaringBitmap();
 
       BitSet reference = new BitSet();
-      bitmap.iterator().forEachRemaining(reference::set);
+      bitmap.forEach(reference::set);
 
-      for (int next : bitmap) {
+      bitmap.forEach(x -> {
```

Comment:
I would keep the variable name `next `instead of `x`, for the sake of smaller diff.